fitstools/smoothing.h still uses TExpr, which was rewritten in LTL 2.0.

diff --git a/fitstools/Makefile.am b/fitstools/Makefile.am
index cca053f..7e6385b 100644
--- a/fitstools/Makefile.am
+++ b/fitstools/Makefile.am
@@ -17,7 +17,6 @@ bin_PROGRAMS = 				\
 	meanfits			\
 	multiplyfits			\
 	rotrevfits			\
-	smoothfits			\
 	statfits			\
 	subtractfits			\
 	sumfits				\
@@ -47,7 +46,6 @@ maskfits_SOURCES	= maskfits.cpp fitstools.h
 meanfits_SOURCES	= meanfits.cpp
 multiplyfits_SOURCES	= multiplyfits.cpp fitstools.h
 rotrevfits_SOURCES	= rotrevfits.cpp fitstools.h
-smoothfits_SOURCES	= smoothfits.cpp smoothing.h fitstools.h
 statfits_SOURCES	= statfits.cpp fitstools.h
 subtractfits_SOURCES  	= subtractfits.cpp fitstools.h
 sumfits_SOURCES	  	= sumfits.cpp fitstools.h
